chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/DeprecatedFieldReviewer.java

```diff
@@ -25,10 +23,8 @@ public DeprecatedFieldReviewer(ErrorMessageUtil msgUtil, FF4j ff4j) {
     @Override
     public void review(CertifiedProductSearchDetails existingListing,
             CertifiedProductSearchDetails updatedListing) {
-        if (!Objects.equals(existingListing.getTransparencyAttestationUrl(), updatedListing.getTransparencyAttestationUrl())) {
-            updatedListing.getWarningMessages()
-            .add(msgUtil.getMessage("deprecated.field.update", "transparencyAttestationUrl", "mandatoryDisclosures"));
-        }
+        //If there are deprecated fields that have been updated between existingListing and updatedListing
+        //add a warning to the updatedListing here using "deprecated.field.update"
```

Comment:
Also missing something. Is this a TODO?

Reply:
No.. I left this comment for future development, although it was before I realized that there would be content left in this reviewer. I didn't want to delete the reviewer entirely because I am sure there will be deprecated fields to put here in the future so just left a comment telling someone what to do when that happens.